app/components/uri/URILandingDialog.js

```diff
@@ -0,0 +1,76 @@
+// @flow
+import React, { Component } from 'react';
+import classnames from 'classnames';
+import { observer } from 'mobx-react';
+import { Button } from 'react-polymorph/lib/components/Button';
+import { ButtonSkin } from 'react-polymorph/lib/skins/simple/ButtonSkin';
+import { defineMessages, intlShape } from 'react-intl';
+
+import Dialog from '../widgets/Dialog';
+import DialogCloseButton from '../widgets/DialogCloseButton';
+import LocalizableError from '../../i18n/LocalizableError';
+
+const messages = defineMessages({
+  titleLabel: {
+    id: 'wallet.send.from.uri.label',
+    defaultMessage: '!!!Title',
+  },
+});
+
+type Props = {
+  onSubmit: Function,
+  onCancel: Function,
+  onCancel: Function,
+  error?: ?LocalizableError,
+};
+
+type State = {
+  warningConfirmed: boolean,
+};
+
+@observer
+export default class URILandingDialog extends Component<Props, State> {
+
+  static defaultProps = {
+    error: undefined,
+    onBack: undefined,
+  }
+
+  submit = () => {
+    this.props.onSubmit();
+  };
+
+  render() {
+    const { onCancel, onBack } = this.props;
+
+    return (
+      <Dialog
+        title="Please read the following information"
+        closeOnOverlayClick={false}
+        closeButton={<DialogCloseButton />}
+        classicTheme={true}
+        onClose={this.props.onClose}
+      >
+        <div>
+          <div>
+            <p>
+              You are about to perform a transaction from a URI link.
```

Comment:
I guess you are going to do localization, here you can refer: https://github.com/Emurgo/yoroi-frontend/blob/f72c1743ebf5658d53a2ce24e1701a5f833b896b/app/components/wallet/hwConnect/ledger/AboutDialog.js#L29